tests/api/conftest.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+from rest_framework.test import APIRequestFactory
+
+from tests.factories import UserFactory, ExternalUserFactory
+
+@pytest.fixture
+def arf():
+    return APIRequestFactory()
+
+
+@pytest.fixture()
+def api_user(transactional_db):
+    e = ExternalUserFactory()
+    return e.owner
```

Comment:
Used for auth. Any authenticated user will do (though may have to update this in future if we start doing more fine-grained permissions)